whelktool/scripts/syslib/Copy_holdings.groovy

```diff
@@ -0,0 +1,45 @@
+String CURRENT_SIGEL = 'https://libris.kb.se/library/Ssb'
+String NEW_SIGEL = 'https://libris.kb.se/library/Mlc'
+
+File bibIDs = new File(scriptDir, "Copy_hold_bibID.txt")
+
+def holdList = []
+
+String bibidstring = bibIDs.readLines().join("','")
+
+selectBySqlWhere("""id in
+
+(
+ select lh.id
+  from
+   lddb lb
+  left join
+   lddb lh
+  on lh.data#>>'{@graph,1,itemOf,@id}' = lb.data#>>'{@graph,1,@id}'
+ where lb.data#>>'{@graph,0,controlNumber}' in ( '$bibidstring' ) and lb.collection = 'bib'
```

Comment:
To be clear: You are sure the list of bib-ids really are "bibids"?

Because @graph,0,controlNumber could look something like 12345 (for older records).

Reply:
The List "bib-IDs" are actually controlnumbers. Maybe that should be more clearly stated in the naming of the file (and possibly the variable). That part of the code is lifted from the move-holds template actually.